test/integration/ext/openai/test_model_openai.py

```diff
@@ -131,34 +110,15 @@ async def test_batch_embed_async():
 @vcr.use_cassette()
 def test_chat():
     e = OpenAIChatCompletion(identifier='gpt-3.5-turbo', prompt='Hello, {context}')
-    resp = e.predict('', one=True, context=['world!'])
+    resp = e.predict_one('', context=['world!'])
 
     assert isinstance(resp, str)
 
 
 @vcr.use_cassette()
 def test_batch_chat():
     e = OpenAIChatCompletion(identifier='gpt-3.5-turbo')
-    resp = e.predict(['Hello, world!'], one=False)
-
-    assert isinstance(resp, list)
-    assert isinstance(resp[0], str)
-
-
-@vcr.use_cassette()
-@pytest.mark.asyncio
-async def test_chat_async():
-    e = OpenAIChatCompletion(identifier='gpt-3.5-turbo', prompt='Hello, {context}')
-    resp = await e.apredict('', one=True, context=['world!'])
-
-    assert isinstance(resp, str)
-
-
-@vcr.use_cassette()
-@pytest.mark.asyncio
-async def test_batch_chat_async():
-    e = OpenAIChatCompletion(identifier='gpt-3.5-turbo')
-    resp = await e.apredict(['Hello, world!'], one=False)
+    resp = e.predict([(('Hello, world!',), {})])
```

Comment:
Do not make sense to run a `batch/many predict` on this way. 

It's better run `e.predict(["hello world"], {})` or `e.predict(["hello world"])` or ``e.predict(["hello world"], n=5)``